pulp_ansible/app/galaxy/v1/utils.py

```diff
@@ -0,0 +1,280 @@
+import datetime
+import glob
+import os
+import subprocess
+import tempfile
+import yaml
+
+
+def get_path_git_root(path):
+    """
+    Find the root directory of a git checkout.
+    """
+    cmd = "git rev-parse --show-toplevel"
```

Comment:
Can we use `git.Repo` for these?

Reply:
Yep, I've actually switched over to that in the task, so this file can probably be completely deleted once I get a better sense of where we are going.